concourse/pipelines/modelbase.py

```diff
@@ -69,10 +70,28 @@ def custom_init(self, raw_dict: dict):
         if raw_dict.get('output_dir', None):
             name = raw_dict['output_dir']
             self.add_output(name + '_path', name + '_path')
+        # normalise step-arguments, if given. For convenience we allow both primitive integers and strings
+        # and lists containing entries of those two types.
+        args = raw_dict.get('arguments', None)
```

Comment:
either call the argument 'args' or name the variable 'arguments' to be more consistent with naming